src/cli/main.py

```diff
@@ -1,9 +1,45 @@
 import requests
 import json
+import argparse
+import os.path
+
+
+parser = argparse.ArgumentParser(description = 'Get weather data from wttr.com'
+                                               ' and save to a json file',
+                                 epilog = 'Enjoy the the weather and the program :)')
+
+parser.add_argument('-city',
+                    '--city',
+                    action='store',
+                    type=str,
+                    default='Miskolc',
+                    metavar='',
+                    help='sets the city for which you want weather data (default Miskolc)')
+parser.add_argument('-saveto',
+                    '--saveto',
+                    action='store',
+                    type=str,
+                    default='weatherdata',
+                    metavar='',
+                    help='set the json output file name (default filename: weatherdata)')
+
+args = parser.parse_args()
+
+city_name = args.city
+json_file = args.saveto+".json"
+
+def check_file_exist():
+    if not os.path.isfile(json_file):
+        create_basic_json_file()
+
+def create_basic_json_file():
+    basic_json_file_structure = {"wttr_requests":[]}
+    with open(json_file, 'w') as new_file:
+        json.dump(basic_json_file_structure, new_file, indent=4)
 
 def read_json_content():
-    with open('weatherdatas.json') as json_file:
-        original_content = json.load(json_file)
+    with open(json_file, 'r') as file:
+        original_content = json.load(file)
     return original_content
 
 def set_new_json_content():
```

Comment:
`append_default_json()`

Reply:
De itt is jó lenne kicsit átgondolni, hogy a függvényed mitől függ és jó dolog-e az amitől jelenleg függ ;)